### ISSUE TITLE
Disables the Android side from using gestures to change the character size of the code ratio on the editor page

### ISSUE THREAD

**Issue by Booklight12** [godotengine/godot#77357]
### Godot version

4.x

### System information

Android

### Issue description

When I slide the slider to locate the line of code I want, the characters always become abnormally small because of miscontact.

### Steps to reproduce

null

### Minimal reproduction project

null

**Comment by Calinou** [comment]
I agree this is error-prone for now, so it makes sense to disable pinch-to-zoom in the script editor.

**Comment by syntaxerror247** [comment]
Yeah, It make sense to me to disable `magnify gesture` in the script editor. An option_button is already available to change the scale.
